helper/testhelpers/teststorage/teststorage.go

```diff
@@ -83,6 +83,14 @@ func MakeRaftBackend(t testing.T, coreIdx int, logger hclog.Logger) *vault.Physi
 	return MakeRaftBackendWithConf(t, coreIdx, logger, nil)
 }
 
+func MakeRaftWithHCLAutopilotBackend(t testing.T, coreIdx int, logger hclog.Logger) *vault.PhysicalBackendBundle {
+	extraConfAutopilotValue := os.Getenv("TEST_AUTOPILOT_HCL_VALUE")
```

Comment:
It feels like this env var could go into a new TestClusterOptions field, say PhysicalFactoryConfig map[string]interface{}.  Then newCore could pass this as a new 4th argument to the PhysicalFactory invocation.

Reply:
Good suggestion. Done.